metagraph/workflows/metagraph_workflows/snakemake/default.yml

```diff
@@ -10,7 +10,6 @@ annotation_labels_source: 'sequence_headers'
 exec_cmd: 'metagraph'
 
 default_disk_mb: 10000
-max_memory_mb: 2000
-
+max_memory_mb: 4048
```

Comment:
Please write kib/mib/... if the base is 1024.

But probably it's better to use kb/mb/... with the base of 1000.
It's simpler for calculations in the mind.

Reply:
snakemake uses `_mb` most likely meaning `mib` (would need to double check). I can change the config keys etc in the workflow  to `_mib` to be explicit what the base is. Using base 1000 seems unorthodox to me.